src/tests/utils/ApiTestSteps.ts

```diff
@@ -374,41 +374,41 @@ export async function getDequeuedSqsMessage(prefix: string): Promise<any> {
 	return getObjectResponse.data;
 }
 
-export async function receiveJwtTokenFromSqsMessage(): Promise<any> {
-	const queueURL = constants.DEV_F2F_IPV_CORE_QUEUE_URL;
-
-	const receiveMessage = () => sqsClient.send(
-		new ReceiveMessageCommand({
-			AttributeNames: ["SentTimestamp"],
-			MaxNumberOfMessages: 10,
-			MessageAttributeNames: ["All"],
-			QueueUrl: queueURL,
-			VisibilityTimeout: 40,
-			WaitTimeSeconds: 20,
-		}),
-	);
-
-	let jwtToken;
-
-	try {
-		const { Messages } = await receiveMessage();
-		for (const m of Messages) {
-			const parsedResponse = JSON.parse(m.Body);
-			const array = Object.values(parsedResponse);
-			jwtToken = array[2];
-			await sqsClient.send(
-				new DeleteMessageCommand({
-					QueueUrl: queueURL,
-					ReceiptHandle: m.ReceiptHandle,
-				}),
-			);
-		}
-	} catch (e: any) {
-		console.error({ message: "got error receiving messages: ", e });
-	}
-
-	return jwtToken;
-}
+// export async function receiveJwtTokenFromSqsMessage(): Promise<any> {
+// 	const queueURL = constants.DEV_F2F_IPV_CORE_QUEUE_URL;
+
+// 	const receiveMessage = () => createSqsClient().send(
+// 		new ReceiveMessageCommand({
+// 			AttributeNames: ["SentTimestamp"],
+// 			MaxNumberOfMessages: 10,
+// 			MessageAttributeNames: ["All"],
+// 			QueueUrl: queueURL,
+// 			VisibilityTimeout: 40,
+// 			WaitTimeSeconds: 20,
+// 		}),
+// 	);
+
+// 	let jwtToken;
+
+// 	try {
+// 		const { Messages } = await receiveMessage();
+// 		for (const m of Messages) {
+// 			const parsedResponse = JSON.parse(m.Body);
+// 			const array = Object.values(parsedResponse);
+// 			jwtToken = array[2];
+// 			await createSqsClient().send(
+// 				new DeleteMessageCommand({
+// 					QueueUrl: queueURL,
+// 					ReceiptHandle: m.ReceiptHandle,
+// 				}),
+// 			);
+// 		}
+// 	} catch (e: any) {
+// 		console.error({ message: "got error receiving messages: ", e });
+// 	}
+
+// 	return jwtToken;
+// }
 
```

Comment:
I have commented the unused function as it required some refactoring for sqsClient. @jackmardell can you checkout this branch and run the test and update me incase you issues. I was unable to run the tests locally by adding the .env file yesterday. Thanks.

Reply:
Hmm so what's going on with this? Do we just not need the functions at all @jackmardell ?

Reply:
Yeah none of the tests use this function, I checked the history and it seems this function was added 7 months ago so honestly it might just be fine to remove it entirely?

Reply:
Ok nice, can someone please remove them then? I have a feeling it has been replaced by the test harness